src/core/localization/translations/en/common.json

```diff
@@ -173,16 +173,16 @@
   "analytics_panel": {
     "header_title": "Analytics",
     "error_loading": "Error while fetching analytics data",
-    "info_short": "Calculations are made for the selected area."
+    "info_short": "Calculations are made for selected area."
   },
   "advanced_analytics_panel": {
     "header_title": "Advanced analytics",
     "error": "Error while fetching advanced analytics data"
   },
   "advanced_analytics_empty": {
-    "not_found": "Sorry, requested event not found.",
-    "error": "An error occured!",
-    "analytics_for_selected": "Analytics for a selected area",
+    "not_found": "Sorry, the requested disaster was not found.",
```

Comment:
@hevanskontur or @gmdowlingkontur can you take a look at this file (except 'about Disaster Ninja' part) and add your comments if there are any mistakes?